packages/components/pill/stories/Pill.stories.tsx

```diff
@@ -205,6 +205,7 @@ export const InSmallContainer: Story<PillInternalProps> = (args) => {
       maxWidth: 200,
     }),
   };
+  const onDrag = args.isDraggable ? args.onDrag : undefined;
```

Comment:
Only pass the `onDrag` callback if the `isDraggable` argument has been adjusted within the story controls. (otherwise the `onDrag` is always truthy)